atlas/src/shared/components/Grid/Grid.tsx

```diff
@@ -25,11 +46,14 @@ const Grid: React.FC<GridProps> = ({ children, className, gap = spacing.xl, repe
       }
     },
   })
+  const isLargeViewport = useMediaQuery({ query: `(min-width: ${LARGE_VIEWPORT_BREAKPOINT})` })
```

Comment:
Why not use a regular CSS media query here, without JS?

Reply:
For some reason it was not working properly when I tried, but now looks fine, so I removed this and went back to regular CSS.